pkg/constants/constants.go

```diff
@@ -172,6 +172,14 @@ func RouteName(name string, verb KFServiceVerb) string {
 	return name + "-" + verb.String()
 }
 
+func VirtualServiceName(name string) string {
```

Comment:
Why suffix the vs with -vs?

Imo it's cleaner to just name all the resources corresponding to our components, with the same base name together. kubectl get "type" "name"

Reply:
removed suffix!